packages/core/src/stylable-resolver.ts

```diff
@@ -437,44 +428,4 @@ export class StylableResolver {
 
         return extendPath;
     }
-    public validateImports(meta: StylableMeta, diagnostics: Diagnostics) {
```

Comment:
It's a public API. We need to make sure no one is using this.

Reply:
Checked with @barak007 when I moved it, but just in case: @barak007 - would you mind checking again?

Reply:
talked with Barak offline - this is not public API outside the library and can be removed.